index.js

```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+		iframe.contentWindow.onresize = null;
```

Comment:
I think the teardown function should detach the iframe instead, otherwise that will be left behind and nothing will make use of it.